cloudflared/rootfs/etc/services.d/cloudflared/run

```diff
@@ -23,7 +23,26 @@ else
     certificate="${data_path}/cert.pem"
     tunnel_name="$(bashio::config 'tunnel_name')"
 
+    # Map HomeAssistant log levels to Cloudflared
+    if bashio::config.exists 'log_level' ; then
+        case $(bashio::config 'log_level') in
+            "trace") cloudflared_log="debug";;
+            "debug") cloudflared_log="debug";;
+            "info") cloudflared_log="info";;
+            "notice") cloudflared_log="info";;
+            "warning") cloudflared_log="warn";;
+            "error") cloudflared_log="error";;
+            "fatal") cloudflared_log="fatal";;
+        esac
+
+        bashio::log.info "Cloudflared log level set to \"${cloudflared_log}\""
+    else
+        cloudflared_log="info"
+    fi
+
     bashio::log.info "Connecting Cloudflared Tunnel..."
     bashio::log.debug "using ${config} config file"
-    exec cloudflared --origincert="${certificate}" --no-autoupdate tunnel --config="${config}" run "${tunnel_name}"
+    
+    exec cloudflared --origincert=${certificate} --no-autoupdate \
+        tunnel --config=${default_config} --loglevel="${cloudflared_log}" run "${tunnel_name}"
```

Comment:
--loglevel property should also be included for the quick-tunnel in line 10, do you agree?
I also just realized that it is also missing the --no-autoupdate for the quick tunnel, so maybe you can fix that as well.